src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecord.cs

```diff
@@ -2,21 +2,64 @@
 using Orchard.Data.Conventions;
 
 namespace Orchard.AuditTrail.Models {
+    /// <summary>
+    /// Audit Trail Event Record in the database.
+    /// </summary>
     public class AuditTrailEventRecord {
         public virtual int Id { get; set; }
+
+        /// <summary>
+        /// The happend time of the event.
+        /// </summary>
         public virtual DateTime CreatedUtc { get; set; }
+
+        /// <summary>
+        /// The user name of the user who made the event..
```

Comment:
Better phrased as:
"The user name of the user whose change caused the event to occur."
And remove double periods at the end.